packages/design-picker/src/components/premium-badge/index.tsx

```diff
@@ -7,10 +7,15 @@ import './style.scss';
 
 interface Props {
 	className?: string;
+	tooltipPosition?: string;
 	isPremiumThemeAvailable?: boolean;
 }
 
-const PremiumBadge: FunctionComponent< Props > = ( { className, isPremiumThemeAvailable } ) => {
+const PremiumBadge: FunctionComponent< Props > = ( {
+	className,
+	tooltipPosition = 'bottom left',
```

Comment:
I never saw we use `tooltipPosition` anywhere. How about just keeping it unchanged?

Reply:
I used it [here](https://github.com/Automattic/wp-calypso/pull/66901/files#diff-98949e427d8352457a31ff332a913b568f4e986561aaf8f2c70de921842c21faR154)! The reason for exposing `tooltipPosition` as a prop is because the default value is "bottom right", which causes a tooltip arrow positioning issue now that the premium badge is on the left side of the meta section.

Reply:
Oops...I misread the default value 🙈